media/css/mozorg/home/home-b.scss

```diff
@@ -0,0 +1,576 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../pebbles/includes/lib';
+@import '../../hubs/buttons';
+
+body {
+    @include open-sans;
+    border-top: 0;
+}
+
+h2,
+h3 {
+    @include zilla-slab;
+}
+
+// hide download button in the nav
+#nav-download-firefox,
+#global-nav-download-firefox {
+    display: none;
+}
+
+// adjust nav spacing to use space freed up by hiding the dl button
+@media #{$mq-tablet} {
+    .moz-global-nav .nav-primary-links {
+        width: calc(100% - 110px);
+    }
+}
+
+@media #{$mq-desktop} {
+    .moz-global-nav .nav-primary-links {
+        width: calc(100% - 220px);
+    }
+}
+
+// flexbox layout for savvy browsers to push nav items to the right
+@supports (display: flex) {
+    @media #{$mq-tablet} {
+        #nav-main-menu,
+        .js #nav-main-menu {
+            @include flexbox;
+            @include justify-content(flex-end);
+        }
+    }
+}
+
+main {
+    overflow: hidden;
+}
+
+
+// *** INTRO *** //
+#intro {
+    background: #e6d4e0 url('/media/img/home/2016/b/hero-bg-mobile.jpg') center center no-repeat;
+    color: #000;
+    position: relative;
+
+    h2,
+    h3 {
+        @include border-box;
+        margin: 0 auto;
+    }
+
+    .content {
+        display: block;
+        padding: 40px 20px;
+        position: relative;
+        width: 100%;
+        z-index: 2;
+    }
+
+    h2 {
+        @include font-size(32px);
```

Comment:
Nit: try to use [font size mixins](https://github.com/mozilla/bedrock/blob/master/media/css/pebbles/includes/mixins/_typography.scss) for consistency, instead of using arbitrary font sizes. Same applies to occurences below

Reply:
Yeah, I was being lazy and copy/pasted from `home.scss`. Will update to use the pre-defined font sizes.